### ISSUE TITLE
Create Articles Detailed Page

### ISSUE THREAD

**Issue by gconnect** [WikiEducationFoundation/apps-android-wikiedudashboard#204]
**Summary:**
There is no detailed page for the articles

**Steps to reproduce:**
Create a detailed Page

Would you like to work on the issue?
Yes.

**Comment by MarinaTaran** [comment]
can I ask what do you mean "detail page for the articles"? we have for each articles on home page some details is that what you mean or not? I would like to work on this task but I didn't understand clearly task!
![Screenshot_1571908483](https://user-images.githubusercontent.com/42735198/67471840-c38f1580-f658-11e9-90b2-ed17b8933f2e.png)

**Comment by gconnect** [comment]
> Oh,if on question "Would you like to work on the issue?" it's written yes it means you work on this issue?:))

If you want to work on it you can get approval from @ujjwalagrawal17

**Comment by ujjwalagrawal17** [comment]
We want to implement and reuse the same article details page of Wikipedia app so that the users have a consistent experience using any of the apps.

**Comment by ujjwalagrawal17** [comment]
@gconnect you can have a look at how Wikipedia App looks, the article details page there.

**Comment by MarinaTaran** [comment]
@ujjwalagrawal17 you mean " the same article details page of Wikipedia app",same  as this details on home page with the same fields? but you want to create a separate, for example,  fragment (maybe or activity) of articles details and when I push on some article I see this detail?

**Comment by ujjwalagrawal17** [comment]
Yes. On clicking a particular article we should be able to see the details of that.

**Comment by MarinaTaran** [comment]
@ujjwalagrawal17 so,can I work on it?

**Comment by ujjwalagrawal17** [comment]
I think It's a big task. Please have a look at the Wikipedia app, let me know if you can implement the same here. For now, we just redirect to Wikipedia app (article detail page) I think there should be an intent for that.

**Comment by MarinaTaran** [comment]
I downloaded Wikipedia app ,can you exactly explain me where I can find it,because I don't see detail page?You mean maybe "about this article"?

**Comment by ujjwalagrawal17** [comment]
![Screenshot_20191118-022914](https://user-images.githubusercontent.com/19607555/69013944-534f8780-09ab-11ea-82cb-947b81f7b9fd.jpg)
![Screenshot_20191118-022908](https://user-images.githubusercontent.com/19607555/69013945-534f8780-09ab-11ea-826f-854408a3c8be.jpg)
![Screenshot_20191118-022859](https://user-images.githubusercontent.com/19607555/69013946-53e81e00-09ab-11ea-9413-9d9f2a7d605c.jpg)

The first and second image are from the articles details page. If possible we would like to reuse them.
